src/components/controls/AudioMeter.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+export interface IAudioMeterRenderer {
+    (audioMonitor: any): JSX.Element | null | false;
+}
+
+export interface IAudioMeterProps {
+    audioMonitor: any;
+    render?: IAudioMeterRenderer
+};
+
+export class AudioMeter extends React.PureComponent<IAudioMeterProps, {}> {
```

Comment:
CustomAudioMeter is quite similar with this one?

What's the difference? Can we share the code?

Reply:
Done